services/coincap/coincap-symbol.service.js

```diff
@@ -0,0 +1,35 @@
+import Joi from 'joi'
```

Comment:
Given this isn't a value that is going to change over time, is the symbol badge actually providing any value?. I'd be inclined to say if people really want this, you could use https://img.shields.io/badge/bitcoin-BTC-green without us having to call an API to get the value, especially given there is a rate limit to consider. I think we should just ditch this one.